websites/P/Pass Culture/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "807326854314590228",
+		"name": "Yuuto"
+	},
+	"service": "Pass Culture",
+	"description": {
+		"en": "The \"pass Culture\" is a system of access to cultural activities set up by the French government for young people between 15 and 18 years old.",
+		"fr": "Le \"pass Culture\" est un dispositif d'accès aux activités culturelles mis en place par le gouvernement français à destination des jeunes entre 15 et 18 ans."
```

Comment:
Strange capitalization. Are these official descriptions? If not, try to use official ones.

Reply:
Yes these are official descriptions.